lib/process-services-cloud/src/lib/form/components/form-cloud.component.ts

```diff
@@ -314,7 +320,7 @@ export class FormCloudComponent extends FormBaseComponent implements OnChanges,
                 formValues[variable.name] = variable.value;
             });
 
-            const form = new FormModel(formCloudRepresentationJSON, formValues, this.readOnly);
+            const form = new FormModel(formCloudRepresentationJSON, formValues, this.readOnly, this.formService);
```

Comment:
Why do we need to pass the formService into the form model at this point?

Reply:
ValueChanged of form field is weirdly triggered into the formfield model please double check also that part as it might create problems bubbling that action up